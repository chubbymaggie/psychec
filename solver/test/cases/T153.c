void f() {
    struct Bar tom;
    tom.bil.jack = 10;
    struct Foz* yum;
    yum->jil->me = 10;
}
