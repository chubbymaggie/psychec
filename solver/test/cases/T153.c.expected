/* Forward declarations */
typedef  struct TYPE_4__   TYPE_2__ ;
typedef  struct TYPE_3__   TYPE_1__ ;

/* Type definitions */
struct Foz {TYPE_2__* jil; } ;
struct TYPE_3__ {int jack; } ;
struct Bar {TYPE_1__ bil; } ;
struct TYPE_4__ {int me; } ;

/* Variables and functions */

